<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyChat</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icons/sphere32.png</normaloff>:/icons/sphere32.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QWidget" name="widget_users" native="true">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_users">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Пользователи</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_refresh_users">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>Обновить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="list_users">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>300</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_send_file">
         <property name="text">
          <string>Передать файл</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_chat" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextEdit" name="text_chat">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>150</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1" rowspan="2">
            <widget class="MyTextEdit" name="text_message">
             <property name="textInteractionFlags">
              <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextEditable|Qt::TextEditorInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="button_copy">
             <property name="text">
              <string>Копировать</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="button_clear">
             <property name="text">
              <string>Очистить</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="button_send">
             <property name="text">
              <string>Отправить</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
   <zorder>widget_chat</zorder>
   <zorder>widget_users</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Действия</string>
    </property>
    <addaction name="action"/>
    <addaction name="separator"/>
    <addaction name="action_Alt_F4"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>О программе</string>
    </property>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
  <action name="action">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="action_Alt_F4">
   <property name="text">
    <string>Выход Alt+F4</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Об авторе</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyTextEdit</class>
   <extends>QTextEdit</extends>
   <header location="global">mytextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
